api/rest.go

```diff
@@ -1449,6 +1448,115 @@ func StoragesHandler(c *gin.Context) {
 	concludeRequest(c, resp, err)
 }
 
+//Labels
+func LabelListHandler(c *gin.Context) {
+	var err *HttpError
+	var resp interface{}
+	if c.Request.Method == http.MethodGet || c.Request.Method == "" {
+		var r LabelsRequest
+		if c.Bind(&r) != nil {
+			return
+		}
+		resp, err = handleGetLabelList(c, c.MustGet("MDB").(*sql.DB), &r)
+	}
+	if c.Request.Method == http.MethodPost {
+		var r CreateLabelRequest
+		if c.Bind(&r) != nil {
+			return
+		}
+		tx := mustBeginTx(c)
+		label, err := handleCreateLabel(c, tx, &r)
+		mustConcludeTx(tx, err)
+
+		if err == nil {
+			emitEvents(c, events.LabelCreateEvent(label))
+		}
+		resp = label.UID
+	}
+	concludeRequest(c, resp, err)
+}
+
+func LabelHandler(c *gin.Context) {
+	id, e := strconv.ParseInt(c.Param("id"), 10, 0)
+	if e != nil {
+		NewBadRequestError(errors.Wrap(e, "id expects int64")).Abort(c)
+		return
+	}
+
+	var err *HttpError
+	var resp interface{}
+	if c.Request.Method == http.MethodGet || c.Request.Method == "" {
+		resp, err = handleGetLabel(c, c.MustGet("MDB").(*sql.DB), id, common.PERM_LABEL_READ)
+	}
+
+	if c.Request.Method == http.MethodPut {
+		if !can(c, secureToPermission(common.SEC_SENSITIVE), common.PERM_LABEL_MODERATE) {
```

Comment:
Why always checking `common.SEC_SENSITIVE` ?
This should be the secure level of the specific unit that the user want to moderate.
Some user can moderate public units and some can (in addition to that) moderate sensitive units.

Reply:
I now see that both `archive_editor` and `archive_label-moderation` have sensitive access.
So no one has anything below that.
Maybe just have a comment saying that we need to change this if we change permission csv config

Reply:
add comment